cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ServerManagementTest.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen.refcard
+
+import org.neo4j.cypher.docgen.RefcardTest
+import org.neo4j.cypher.docgen.tooling.DocsExecutionResult
+import org.neo4j.cypher.docgen.tooling.QueryStatisticsTestSupport
+import org.neo4j.graphdb.Transaction
+
+class ServerManagementTest extends RefcardTest with QueryStatisticsTestSupport {
+  val graphDescription = List()
+  val title = "(★) Server management"
+  override val linkId = "administration/servers"
+
+  override def assert(tx:Transaction, name: String, result: DocsExecutionResult): Unit = {}
+
+  def text = """
+###dontrun
+// Can't be run since there is no cluster available.
```

Comment:
well... the `//` is what was required for the test to work at all, so I'm happy with keeping the comment as well instead of just having empty comment after every other